### ISSUE TITLE
[Prism Client] option to disable pino log

### ISSUE THREAD

**Issue by hieucd04** [stoplightio/prism#1897]
## Summary

I followed instructions here to setup Prism Client: https://meta.stoplight.io/docs/prism/ZG9jOjE2MDY1Njcw-prism-client 
It worked pretty well except for one small thing which was the pino logs outputs to console.

Is there anyway to turn off the pino logs generated by Prism Client?

**Comment by philsturgeon** [comment]
@hieucd04 I’ve pinged this over to the right people to take a look, but from what I remember working on prism, this HTTP Client use case was being removed as we no longer use it for anything and it was never particularly fleshed out.

Do you mind if I ask what your use case is? How is http client being used and where?

**Comment by radzserg** [comment]
@hieucd04 try this treak
```
const client = createClientFromOperations(operations, {
   ... ,
   logger:  pino({
    enabled: false,
  })
});
```
See examples here
https://github.com/stoplightio/prism/pull/1892/files#diff-532a9bcfdd78a801a736f1e4b8fa7b37f179632c5caa36865b5b51d5441ad2b0R27
https://github.com/stoplightio/prism/pull/1892/files#diff-e8eed100de477e08ab55310efa4f1f5e1ce9b5e56049ce508a8964fd07a952a9R8

**Comment by hieucd04** [comment]
I managed to get it to work using your examples.
Thank you!

@philsturgeon 
I use it for `automated API contract testing`.
I wrote a Node.JS script which sends a bunch of pre-defined HTTP requests to my API either directly with `Prism HTTP Client` or via `Prism Validation Proxy`. The problem with validation proxy approach is that it doesn't seem to be flexible enough for my use case:

1. As a developer, I want to send invalid HTTP requests to my API to test how it handles them
2. As a developer, I want to turn off `request validation` in Prism Validation Proxy. So that `sl-violations` header doesn't contain request validation errors. I don't care about request validation errors. I just need response validation errors.
3. As a developer using TypeScript, I want `Prism response` to be TypeScript object with type information associated with it. So that, it makes my life easier.
4. As a developer, I want to tell if `Validation Proxy / Prism HTTP Client`'s response is a mock response from OpenAPI document or real response from my API. So that, my test script can react accordingly.

Given those requirements, `Prism HTTP Client` looks like a better candidate, for now.
